slsim/image_simulation.py

```diff
@@ -323,41 +342,49 @@ def point_source_image_at_time(
     :param mag_zero_point: magnitude zero point in band
     :param delta_pix: pixel scale of image generated
     :param num_pix: number of pixels per axis
-    :param psf_kernels: psf kernels for the given exposure.
+    :param psf_kernel: psf kernel for the given exposure.
+    :param transform_pix2angle: transformation matrix (2x2) of pixels into coordinate
+        displacements
     :param time: time is an image observation time [day].
     :return: point source images with variability
     """
 
-    image_data = point_source_coordinate_properties(
-        lens_class=lens_class,
-        band=band,
-        mag_zero_point=mag_zero_point,
-        delta_pix=delta_pix,
-        num_pix=num_pix,
-        transform_pix2angle=transform_pix2angle,
-    )
-
+    kwargs_model, kwargs_params = lens_class.lenstronomy_kwargs(band="i")
```

Comment:
this is a hard-coded i-band. Is this needed? If you do not need a band, I would not require you to have a certain band be present